students/TinaB/lesson08/test_circle.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python3
+"""Lesson 08 Unit Tests for Circle CLass Assignment - """
+
+import unittest
+from math import pi
+import circle
+
+
+class TestCircle(unittest.TestCase):
+
+    def __init__(self, *args, **kwargs):
+        super(TestCircle, self).__init__(*args, **kwargs)
+        self.t_radius = 4
+        self.t_diameter = 2 * self.t_radius
+        self.t_area = pi * self.t_radius**2
+        self.t_perimeter = 2 * self.t_radius * pi
```

Comment:
You don't want to inherit from the `TestCase` class to set up test data in this way, instead you can create a circle instance for testing in the `setUp` method rather